### PR TITLE
TRAFODION-2823 some test case donot close connection

### PR DESCRIPTION
**Pull request by gpj1987**
some classes in incubator-trafodion/dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test do not close statement and connection, which will lead to no connection can be use.
classes include : 
PropTest.java
TestNetworkTimeout.java
TestWrap.java

### CODE REVIEW

File: dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestWrap.java [hunk 1]
```diff
@@ -51,7 +51,11 @@ public void testIsWrapFor() {
             conn = Utils.getUserConnection();
             boolean result = conn.isWrapperFor(Connection.class);
             assertTrue("It is wrapper for this interface", result);
+            conn.close();
         } catch (SQLException e) {
+            if (conn !=null) {
+                conn.close();
```
Comment (DaveBirdsall) [inline dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestWrap.java]: The Jenkins builds are complaining that this code has an uncaught exception of some sort. Could you take a look? Thanks.

**Comment by selvaganesang** [comment]
jenkins, retest

**Comment by DaveBirdsall** [comment]
The JDBC test suite is failing with a compilation error:

2017-12-13 19:11:08 [ERROR] /home/jenkins/workspace/jdbc_test-cdh/trafodion/dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestNetworkTimeout.java:[75,36] error: not a statement
2017-12-13 19:11:08 [ERROR] /home/jenkins/workspace/jdbc_test-cdh/trafodion/dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestNetworkTimeout.java:[75,37] error: ';' expected

Please fix and we'll try again.

File: dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestNetworkTimeout.java [hunk 2]
```diff
@@ -64,10 +72,19 @@ public void testGetNetworkTimeout() {
 		Connection conn = null;
 		try {
 			System.out.println("Connecting to database...");
-			conn = Utils.getUserConnection();
+			conn = Utils.getUserConnection();S
```
Comment (DaveBirdsall) [inline dcs/src/test/jdbc_test/src/test/java/org/trafodion/jdbc_test/TestNetworkTimeout.java]: The extra "S" at the end of the line appears to be causing the JDBC build failure

**Pull request merged**